src/toil/leader.py

```diff
@@ -50,15 +52,54 @@ def __init__(self, jobStore):
                               args=(jobStore, self._stop))
         self._worker.start()
 
-    @staticmethod
-    def statsAndLoggingAggregator(jobStore, stop):
+    @classmethod
+    def writeLogFiles(cls, jobNames, jobLogList, config):
+        def createName(logPath, jobName, logExtension):
+            logName = jobName.replace('-', '--')
+            logName = logName.replace('/', '-')
+            logName = logName.replace(' ', '_')
+            logName = logName.replace("'", '')
```

Comment:
Spaces, single and double quotes are actually allowed in filenames but require escaping when you want to refer to such filenames in a shell. So I understand the motivation for this.